modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/SymbolMBLayer.java

```diff
@@ -1047,6 +1051,15 @@ public AnchorPoint anchorPoint() {
         }
         return sf.anchorPoint(ff.literal(anchor.getX()), ff.literal(anchor.getY()));
     }
+    
+    /**
+     * 
+     * @return True if the layer has a text-max-angle explicitly provided.
+     */
+    public boolean hasTextMaxAngle() throws MBFormatException {
+        return parse.isPropertyDefined(layout, "text-max-angle");
+    }
```

Comment:
For future reference, most of the surrounding code was core API - namely the getters for the actual parameters in the MBStyle spec, hence why it was public. This is more of a utility method, so doesn't need to be public.